src/OrleansSQLUtils/Storage/DbConnectionFactory.cs

```diff
@@ -16,6 +16,13 @@ private static CachedFactory GetFactory(string invariantName)
             // Seeks for database provider factory classes from GAC or as indicated by
             // the configuration file, see at <see href="https://msdn.microsoft.com/en-us/library/dd0w4a2z%28v=vs.110%29.aspx">Obtaining a DbProviderFactory</see>.       
 
+            if (invariantName == AdoNetInvariants.InvariantNamePostgreSql)
```

Comment:
This isn't different than the way we currently load providers. We can decide to change that, but currently we only load providers that are in the GAC. (SQL Server and MySQL)